crates/re_renderer/src/line_strip_builder.rs

```diff
@@ -188,6 +188,15 @@ impl<'a> LineBatchBuilder<'a> {
 
     /// Adds a 3D series of line connected points.
     pub fn add_strip(&mut self, points: impl Iterator<Item = glam::Vec3>) -> LineStripBuilder<'_> {
+        if self.0.strips.len() >= LineDrawData::MAX_NUM_STRIPS {
+            re_log::error_once!(
+                "Reached maximum number of supported line strips of {}.
+     See also https://github.com/rerun-io/rerun/issues/957",
```

Comment:
This will log:

```
[12:32:00] Reached maximum number of supported line strips of 42.
     See also https://github.com/rerun-io/rerun/issues/957
```

Better to keep it on one line imho
```suggestion
                "Reached maximum number of supported line strips of {}. \
                 See also https://github.com/rerun-io/rerun/issues/957",
```